source/Extractors/include/ExtractorFCCSW.h

```diff
@@ -60,6 +60,7 @@ class ExtractorFCCSW : public AnalyzerUnit {
   std::string printWithUnit(double value, int precision, std::string unit);
 
   std::unique_ptr<tinyxml2::XMLDocument> m_xmlDoc;      //!< An XML base container (pointing to XML file created on disc)
+  std::unique_ptr<tinyxml2::XMLDocument> m_xmlDefinitionsDoc;      //!< An XML base container (pointing to XML file created on disc)
```

Comment:
I'd maybe add comment on the difference between m_xmlDoc // XML container pointing to an XML file describing detailed geometry structure & m_xmlDefinitionsDoc // XML container pointing to an XML file describing base geometry components or something similar ....